MobSF/settings.py

```diff
@@ -225,6 +229,9 @@
 # https://docs.djangoproject.com/en/dev/howto/static-files/
 STATIC_URL = '/static/'
 
+# 500MB
+DATA_UPLOAD_MAX_MEMORY_SIZE = 524288000

```

Comment:
Can you keep this to `256 MB` as we have that followed elsewhere.

Reply:
Should I change it to 256MB or delete it?

Reply:
Please change that to equivalent of 256 MB